drivers/adc/ad7768-1/ad77681.c

```diff
@@ -272,39 +272,42 @@ uint8_t ad77681_get_frame_byte(struct ad77681_dev *dev)
 int32_t ad77681_spi_read_adc_data(struct ad77681_dev *dev,
 				  uint8_t *adc_data)
 {
-	uint8_t crc_calc_buf[4], buf[6], crc, frames_8byte = 0;
```

Comment:
You should have maximul of 72 characters per commit line.

Reply:
Done with the edits of the commit message.